airbyte-integrations/connectors/destination-gcs/sample_secrets/config.json

```diff
@@ -6,5 +6,8 @@
     "credential_type": "HMAC_KEY",
     "hmac_key_access_id": "<access-id>",
     "hmac_key_secret": "<secret>"
+  },
+  "format": {
```

Comment:
This is not related to this change.

Reply:
This is not related to this change, but I think the creator missed to include the format in "config.json" file. I waste some time to figure out the sample "config.json" has not the format field which is necessary. It give us some strange error message if you do not include the "format" field. In a word, I think it is better to include "format" field in "config.json" file.